projects/asthma-copd/nbs/30_gls_phenoplier/05-summarize.ipynb

```diff
@@ -0,0 +1,2293 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    ):
</pre><p>unnecesary whitespace here maybe?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/phenoplier/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>Yes, there are some aspects of the automatic code formatted that could be improved. Here I use <code><a href="https://github.com/psf/black" target="_blank">black</a></code> to automatically format the source code, which is PEP 8 compliant. So I just stick to whatever <code>black</code> suggests. </p>